config/formats.ts

```diff
@@ -1953,8 +1953,8 @@ export const Formats: FormatList = [
 		name: "[Gen 7] CAP",
 		threads: [
 			`&bullet; <a href="https://www.smogon.com/forums/threads/3621207/">CAP Metagame Discussion</a>`,
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3626018/">CAP Viability Rankings</a>`,
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3648521/">CAP Sample Teams</a>`,
+			`&bullet; <a href="https://www.smogon.com/forums/posts/8691482">CAP Viability Rankings</a>`,
+			`&bullet; <a href="https://www.smogon.com/forums/posts/8691484">CAP Sample Teams</a>`,
```

Comment:
Can you change these all to say `USM CAP`?

Also, can you re-add the slash to the ends of the URLs if not too much trouble